test/PowerShellEditorServices.Test/Language/LanguageServiceTests.cs

```diff
@@ -78,9 +78,18 @@ public async Task LanguageServiceCompletesCommandFromModule()
                     CompleteCommandFromModule.SourceDetails);
 
             Assert.NotEmpty(completionResults.Completions);
+
             Assert.Equal(
-                CompleteCommandFromModule.ExpectedCompletion,
-                completionResults.Completions[0]);
+                CompleteCommandFromModule.ExpectedCompletion.CompletionText,
+                completionResults.Completions[0].CompletionText
+            );
+
+            Assert.Equal(
+                CompleteCommandFromModule.ExpectedCompletion.CompletionType,
+                completionResults.Completions[0].CompletionType
+            );
+
+            Assert.NotNull(completionResults.Completions[0].ToolTipText);
```

Comment:
Note, the signature of `Get-Random` changed so I'm asserting on less things but I think it's fine.